src/engine/game/gui/ingame/equip.h

```diff
@@ -18,13 +18,23 @@
 #pragma once
 
 #include "../gui.h"
+#include "../../../gui/control/button.h"
+#include "../../../gui/control/imagebutton.h"
+#include "../../../gui/control/label.h"
+#include "../../../gui/control/listbox.h"
 
 namespace reone {
 
 namespace game {
 
 class Creature;
 
+class EquipmentBindingHelper {
```

Comment:
Yeah, let's make it part of InGameMenu, no interfaces. Would fit better with the current architecture, if anything:

```cpp
class InGameMenu;

class Equipment:
public:
    Equipment(Game &game, InGameMenu &inGameMenu) : _game(game), _inGameMenu(inGameMenu) {
    }

private:
    Game &_game;
    InGameMenu &_inGameMenu;
```